doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -2,8 +2,9 @@
 ## 11.0.0
 - [General] **Attention:** Added native php types for argument types, property types, return types and strict type declaration where possible. 
  This results in **various bc breaks**. Please make sure to add the corresponding types to your implementation.
+- [Symfony] Bumped Symfony packages to "^6.1". Pimcore 11 will only support Symfony 6.
+- [PHP] The minimum supported PHP version is now 8.1 and added support for 8.2 
```

Comment:
I think you should change the PHP requirement in `composer.json` from `^8.1` to `~8.1.0 || ~8.2.0` if you don't plan (or don't know yet) to support PHP 8.3.

Reply:
Since PHP 8.3 is still in work-in-progress, nobody knows if it will be supported, but it would be safe to assume it will be supported by Pimcore (like a "until proved otherwise" 😄) and that we could later restrict maximum to 8.2 in a bug fix release like https://github.com/pimcore/pimcore/pull/14180

Reply:
As safe as it was to assume that Pimcore 10 would support PHP 8.2? ;)

The problem is: if you have a release (let's say `v11.0`) that states it supports PHP 8.3, composer will install it. If you later disallow this PHP version in say `v11.1` and someone tries to install Pimcore 11 via `composer require pimcore/pimcore` while running on PHP 8.3, composer will install `v11.0` (because `11.1` doesn't support the PHP version, but `v11.0` does (or at least says it does)). Now, if we are in a situation like right now, where you decided to support PHP 8.3 only with the next mayor version Pimcore 12, but it is not released yet, people will get a "broken" version, and maybe it runs, but only a few features are broken that aren't used very often, so they won't notice immediately. That's a big source of frustration.

Reply:
@jdreesen thanks for the input, you're right, `~8.1.0 || ~8.2.0` probably makes more sense. I'd go for it as it is more safe. Thanks!